### PR TITLE
New Ping Pong Gun Model

### PR DESCRIPTION
**Pull request by imgntn**
This PR updates the model URL for the ping pong gun in a variety of scripts.

To test:  

1. Run this script and shoot the ping pong ball gun.
https://rawgit.com/imgntn/hifi/newponggun/examples/toybox/ping_pong_gun/createPingPongGun.js

### CODE REVIEW

**Comment by imgntn** [comment]
https://app.asana.com/0/46580934041217/95931178691851

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d82dcb5333d45ac56cf940d281a6258891815e4 available at: https://gist.github.com/89cb2f49abb8170d1eb6<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7260.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7260.dmg

File: examples/toybox/ping_pong_gun/createPingPongGun.js [hunk 1]
```diff
@@ -11,53 +11,63 @@
 /*global MyAvatar, Entities, AnimationCache, SoundCache, Scene, Camera, Overlays, HMD, AvatarList, AvatarManager, Controller, UndoStack, Window, Account, GlobalServices, Script, ScriptDiscoveryService, LODManager, Menu, Vec3, Quat, AudioDevice, Paths, Clipboard, Settings, XMLHttpRequest, randFloat, randInt */
 Script.include("../../libraries/utils.js");
 
-var scriptURL = Script.resolvePath('pingPongGun.js');
+var scriptURL = Script.resolvePath('pingPongGun.js?' + Math.random());
```
Comment (sethalves) [inline examples/toybox/ping_pong_gun/createPingPongGun.js]: did you intend to leave the random call here?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 567039ef821132c78ad865e60a9cab71364817c8 available at: https://gist.github.com/89cb2f49abb8170d1eb6<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7260.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7260.dmg

**Pull request merged**